src/components/visualizations/proptypes/execution.ts

```diff
@@ -0,0 +1,299 @@
+// (C) 2007-2018 GoodData Corporation
+import * as PropTypes from 'prop-types';
+
+const identifierPropTypes = PropTypes.string;
```

Comment:
As I said before, I'd remove all the React prop types as there's IV components are going to exist only within the TS world, where static type checking should be enough. That means we would need to maintain all this code going forward.

Reply:
I removed all others, but kept these which were imported in react-components. I did not want to touch this code (https://github.com/gooddata/gooddata-react-components/pull/248/files#diff-1c29878a525a2d23600663aee398eed3L9)